AutoHeightImage.js

```diff
@@ -13,16 +13,16 @@ import { NOOP, DEFAULT_HEIGHT } from './helpers';
 // remove `resizeMode` props from `Image.propTypes`
 const { resizeMode, ...ImagePropTypes } = AnimatableImage.propTypes;
 
-function AutoHeightImage(props) {
-  const {
-    onHeightChange,
-    source,
-    width,
-    style,
-    maxHeight,
-    onError,
-    ...rest
-  } = props;
+function AutoHeightImage({
+  source,
+  width,
+  style,
+  onError,
+  maxHeight = Infinity,
+  onHeightChange = NOOP,
+  animated = false,
```

Comment:
you need to manually add animated to the ImageComponent on line 72.

Thank you for this PR I founded and needed to patch mine for now.

Reply:
that is a short term fix, to do it correctly we need to resolve the issue and use function default params assignment as recommended from facebook team